packages/xod-client/src/debugger/messages.js

```diff
@@ -48,3 +48,26 @@ export const DO_NOT_USE_TETHERING_INTERNET_IN_BROWSER = {
   solution:
     'Install the desktop version of the XOD IDE to reveal all features of XOD',
 };
+
+export const logCopyDone = tabLabel => ({
+  title: `${tabLabel} log is copied to the clipboard`,
+});
+export const logCopyError = (tabLabel, error) => ({
+  title: `Can not copy the ${tabLabel} log`,
+  note: error.message,
+  solution: 'Please, report the issue to the XOD developers ',
+});
+export const logCopyNotSupported = () => ({
+  title: 'Your browser does not support a copy log feature',
+  solution:
+    'Try to install the desktop IDE or use the latest version of Google Chrome.',
+});
+
+export const logSaveDone = tabLabel => ({
+  title: `${tabLabel} log is saved on your filesystem`,
+});
```

Comment:
:bug: In desktop IDE this message appears before the file save dialog pops up, even if I choose to cancel (in the future). If it’s hard to make the message appear in the right moment (after the dialog), it’s OK to not show it at all.